texts/en_US.properties

```
@@ -48,6 +48,8 @@ geyser.bootstrap.no_supported_protocol=Your server software does not support the
 geyser.bootstrap.unsupported_java.header=YOU ARE RUNNING AN UNSUPPORTED VERSION OF JAVA\!
 geyser.bootstrap.unsupported_java.message=Geyser will not work on your version of Java in the future. Please update to Java {0}. Current Java version: {1}
 geyser.bootstrap.unsupported_proxy=Your {0} proxy is outdated!
+geyser.bootstrap.unsupported_proxy_backend=This server appears to be running behind a {0} proxy! Geyser should only be installed on the proxy.
+geyser.bootstrap.setup_guide=Please visit the setup guide {0} for help with configuring Geyser.
```

```suggestion
geyser.bootstrap.unsupported_proxy_backend=This server appears to be running behind a {0} proxy! Geyser should only be installed on the proxy.
geyser.bootstrap.setup_guide=Please visit the setup guide for help with configuring Geyser: {0}
```
personally i find links in the middle of a sentence less easy to read